Stripe/STPBankAccount.m

```diff
@@ -127,7 +124,7 @@ + (NSArray *)requiredFields {
              ];
 }
 
-+ (instancetype)decodedObjectFromAPIResponse:(NSDictionary *)response {
++ (nullable instancetype)decodedObjectFromAPIResponse:(nullable NSDictionary *)response {
```

Comment:
Wait, maybe I misunderstand your initial comment? Status is already listed as a required field. The other two are marked as nullable in the header, so I think this is correct already?